pom.xml

```diff
@@ -38,13 +38,13 @@
     </modules>
 
     <properties>
-        <revision>3.1.0</revision>
+        <revision>3.0.0</revision>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <java.version>8</java.version>
+        <java.version>20</java.version>
```

Comment:
Maybe 21?

Reply:
Version `21` did not work, due to a `mockito` dependency. Turned to version `20` and we will conduct further investigation.